docs/site/home.js

```diff
@@ -20,6 +22,20 @@ angular
       templateUrl: 'site/home.html',
       controller: 'HomeCtrl'
     });
+
+    markdownConverterProvider.config({
+      extensions: [function () {
+        return [{
+          type: 'lang',
+          regex: '\\[([^\\]]+)\\]\\((\\/[^)]+)\\)',
```

Comment:
I don't think so, calling `toString()` on `RegExp` will include the trailing/leading slashes. Then Showdown will pass that string to the `RegExp` constructor and that will automatically escape things that shouldn't be escaped.